scipy/stats/_continuous_distns.py

```diff
@@ -940,6 +940,7 @@ def _ppf(self, q, c, d):
         return (q**(-1.0/d) - 1)**(-1.0/c)
 
     def _stats(self, c, d):
+        c, d = np.broadcast_arrays(c, d)
```

Comment:
Currently, `_stats` will only be called with both `c` and `d` being scalars (from `moment()`), or with both `c` and `d` being 1-d arrays of the same length (from `stats()`), so this call of `broadcast_arrays` should  not be necessary.  I think the only effect that you count on is that after this call, `c` is a numpy array (possibly with shape `()`), so you can use the `ndim` attribute below.  If we change that to `np.ndim(c)`, we can remove this call of `broadcast_arrays`.
```suggestion
```